package/gluon-status-page/javascript/status-page.js

```diff
@@ -112,6 +112,17 @@
 		'bytes': function(bytes) {
 			return prettyBytes(bytes);
 		},
+		'neighbour': function(addr) {
+			for (var i in interfaces) {
+				var iface = interfaces[i];
+				var neigh = iface.get_neigh(addr);
```

Comment:
Maybe improvide readability like this, you're not doing any error handling at this point anyway.
```
			for (var interface in interfaces) {
				var neigh = interfaces[interface].get_neigh(addr);
```

Which leads me to the question: How does it look when the node is offline?